Cue/app/tabs/library/LibraryHome.js

```diff
@@ -66,16 +66,18 @@ class LibraryHome extends React.Component {
 
   constructor(props: Props) {
     super(props)
-
+
+    let refreshing = this.props.decks ? false : true;

```

Comment:
setting `refreshing` here will cause it to stay `true` on login. Need a way to set it to `false` once the initial `loadLibrary` after login is done.